### ISSUE TITLE
Java Norm

### ISSUE THREAD

**Issue by FauconFan** [Heijix/nut-lang#30]
We must have a uniform norm for Java, because it's a very verbose language.

I usually use [uncrustify](https://github.com/uncrustify/uncrustify). It works very well on Java, and on C. But it deals with Java Norm here.

I usually use this as a config file for uncrustify:

[config_Java.txt](https://github.com/Heijix/nut-lang/files/2248825/config_Java.txt)

It should be self explained, but if you want the documentation of a uncrustify config file, go [there](https://github.com/uncrustify/uncrustify/blob/master/etc/ben.cfg)

**Comment by xavitator** [comment]
It is very sympathetic. It clears for tabulation @elusyo. However i am for non-return before {. It is just by habit, i know.

**Comment by maiste** [comment]
I agree with @xavitator. In Java style it's more common to use this style. But you decide, I've no problem with both style.
